zmslayout/scss/themes/zms/form-input-counter.scss

```
@@ -28,15 +28,22 @@
             background-color:       #ffffff !important;
             height:                 2.1em;
             font-weight:            bold;
+            width:                  3em;
+            padding-left:           1em;
+            // Hide native steppers if type=number is used anywhere
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)